src/components/user/registerModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { useState } from "react";
+import Form from "react-bootstrap/Form";
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
+import Button from 'react-bootstrap/Button';
+import InputGroup from 'react-bootstrap/InputGroup';
+import Row from 'react-bootstrap/Row';
+
+import { User } from '../../interface/userInterface';
+import { registerUser } from '../../utils/user/TodoApiService';
+import { useTokenContext } from '../../App';
+
+declare function require(name:string);
+var I18n = require('react-redux-i18n').I18n;

```

Comment:
How about the below ?

```suggestion
var t= require('react-redux-i18n').I18n.t;

```